content/contact.md

```diff
@@ -1,7 +1,12 @@
-+++
-title = 'Contact'
-date = 2024-09-26T22:35:27+12:00
-+++
+---
+title: Contact
+featured_image: ''
+omit_header_text: true
+description: We'd love to hear from you
+type: page
+menu: main
```

Comment:
I'll admit I don't know what all these do, but this seems to be responsible for putting the thing in the menu at top!